generate_self_schema.py

```diff
@@ -23,6 +24,7 @@
     UnionType = Union[TypingUnionType, TypesUnionType]
 
 except ImportError:
+    TypesUnionType = None
```

Comment:
Maybe better to do TypesUnionType = TypingUnionType, since there is an `or origin is TypesUnionType` below, where I don't think you want `origin is None` to be acceptable?

Reply:
```suggestion
    TypesUnionType = TypingUnionType
```

Reply:
Thanks